packages/components/icon/src/icon.ts

```diff
@@ -54,29 +52,32 @@ export class Icon extends LitElement {
   }
 
   /**
-   * store all icons from the IconLibrary of the theme in the icon registry for easy access
+   * Store all icons from the IconLibrary of the theme in the icon registry for easy access.
```

Comment:
Do we want to show here for the documentation purposes what exactly `IconLibrary` interface contains?

Reply:
I think the whole process of setting up/loading the icons will need to be described separately. I don't think we can get the full explanation in this column